modules/upgrade.adoc

```diff
@@ -21,7 +21,9 @@ All Kubernetes objects and PVs in each {product-title} cluster are backed up as
 
 [NOTE]
 ====
-The upgrade starting time might be later than the scheduled time. It could be sometimes an hour or longer.
+During a scheduled upgrade policy, an upgrade might take an hour or more to start, regardless of whether it is an immediate upgrade. 
```

Comment:
Maybe consider a revision. At the very least I think this info should be on one light (though slightly revised when doing that). I came up with 

When following a scheduled upgrade policy, there might be a delay of an hour or more before the upgrade process begins, even if it is an immediate upgrade. Additionally, the length of the upgrade might vary based on your workload configuration and could take longer than expected.

Reply:
Have changed according to the suggestion.